constants/constructor-args.ts

```diff
@@ -367,6 +367,13 @@ export const RewardTokenArgs = {
     },
 };
 
+export const ChipsArgs = {
+    TESTNET: {
+        _token: '0xD73EbC44643B594e44Ab46202590458c9606A709',
```

Comment:
Create a constant here, to be reused in the : constants/deployments/deployments-g7-testnet.ts